source/includes/_custom-field.md

```diff
@@ -47,9 +47,47 @@ curl  \
     -H "X-Killbill-ApiKey: bob" \
     -H "X-Killbill-ApiSecret: lazar" \
     -H "Accept: application/json" \
-    "http://127.0.0.1:8080/1.0/kb/customFields/pagination"
+    "http://127.0.0.1:8080/1.0/kb/customFields/pagination?offset=0&limit=100&audit=NONE"
```

Comment:
I believe the `offset` and `limit` parameters are not mandatory. Is there any specific reason for adding them to the URL?